src/store/token.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+export async function getToken(token) {
+	const tokenValue = await ((typeof (token) === 'function')
+		? token()
+		: token);
+
+	return await new Token(tokenValue);
+}
+
+export function compareTokens(a, b) {
+	return a.toString() === b.toString();
+}
+
+export const TOKEN_COOKIE = -1;
+export const TOKEN_COOKIE_CACHE_KEY = 'cookie';
+
+class Token {
+	constructor(token) {
+		this._cacheKey = this._praseCacheKey(token);
+		this._value = token;
+	}
+
+	get value() {
+		return this._value;
+	}
+
+	toString() {
```

Comment:
How often is this used? Is this just for debugging?

Reply:
So this method is called if you use this object as an index. Allows you to do `store = { token: test }` 

Then I used maps...

Reply:
This is also useful since it is a unique string that is the same for each token from this user.